### PR TITLE
CIRC-4237 - IRONdb 0.18.4 Changelog

### PR DESCRIPTION
**Pull request by pamaddox**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+ * Implement histogram /fetch transforms: `{inverse_,}{quantile,percentile}`
+   and `count_{above,below}`.
+ * Bug: Fix crashes related to bad locking when adding/removing a metric locator
+   from the surrogate cache.
+ * Bug: Fix potential integer overflow glitch when using the `/fetch` endpoint
+   that could cause occasional incorrect results.
+ * CAQL: Fix account_id handling for histogram summary views
+ * CAQL: Add sensible default labels to histogram:percentile() output
+ * CAQL: Performance improvements to integrate() function
+ * CAQL: Leverage /fetch endpoint for find() operations
```
Comment (esproul) [inline changelog.md]: I believe this is a substantial performance improvement? If so, we should call that out.

Comment (pamaddox) [reply changelog.md]: No idea if it is or not.

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## Changes in 0.18.4
+2019-10-16
+
+ * Support tailing \*\* in graphite queries in a way that is leaf-only.
+ * Support a filter config option for the monitor module.
+ * Implement topk as a /fetch reducer.
```
[the remark this answers is not in the document]
Comment (pamaddox) [reply changelog.md]: I don't know what "topk" is - I was just copying from the changelog.

[the remark this answers is not in the document]
Comment (esproul) [reply changelog.md]: As I understand it, it's a statistical problem/function that finds the top "k" values from a given set, hopefully efficiently.  I've seen it rendered as "top-k" but I think also "top(k)" in some internal discussions.

**Pull request merged**